idlpy/src/types.c

```diff
@@ -146,13 +179,21 @@ emit_field(
 
     char *type = typename(ctx, type_spec);
 
+    //printf("emit_field name : %s , type %s \n", name , type);
+
     if (idl_is_default_case(parent)) {
+
+        //printf("emit_field idl_is_default_case name : %s , type %s \n", name , type);
```

Comment:
Please don't leave commented code in the PR.

Reply:
ha ha ha, it was an oversight, we have already deleted this printf and another one that we have also found